test/e2e/scale_test.go

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/utils/ptr"
+	capi_e2e "sigs.k8s.io/cluster-api/test/e2e"
+	"sigs.k8s.io/cluster-api/test/framework"
+	"sigs.k8s.io/cluster-api/test/framework/clusterctl"
+	. "sigs.k8s.io/cluster-api/test/framework/ginkgoextensions"
+)
+
+const (
+	scaleClusterControlPlaneEndpointIPPlaceholder   = "scale-cluster-controlplane-endpoint-ip-placeholder"
+	scaleClusterControlPlaneEndpointPortPlaceholder = "scale-cluster-controlplane-endpoint-Port-placeholder"
+	scaleClusterVSphereServerPlaceholder            = "scale-cluster-vsphere-server-placeholder"
+	scaleClusterVSphereTLSThumbprintPlaceholder     = "scale-cluster-vsphere-tls-thumbprint-placeholder"
+	scaleClusterNamePlaceholder                     = "scale-cluster-name-placeholder"
+)
+
+var _ = Describe("When testing the machinery for scale testing using vcsim provider [vcsim] [supervisor] [ClusterClass]", func() {
+	const specName = "quick-start-cluster-class-scale" // prefix must be uniq because this test re-writes the clusterctl config.
```

Comment:
```suggestion
	const specName = "scale-quick-start-cluster-class" // prefix must be uniq because this test re-writes the clusterctl config.
```
Or maybe just scale? (I assume it's easier to find this test in artifacts later if the spec name starts with scale)